core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -586,12 +592,28 @@ private[ui] class JobPagedTable(
   override def row(jobTableRow: JobTableRowData): Seq[Node] = {
     val job = jobTableRow.jobData
 
+    val killLink = if (killEnabled) {
```

Comment:
Nice catch, since it's supposed to be html I'll add a `else Seq.empty` to each instance of this